google-cloud-storage/src/main/java/com/google/cloud/storage/UnifiedOpts.java

```diff
@@ -1623,6 +1639,16 @@ public String toString() {
     }
   }
 
+  static final class OverrideUnlockedRetention extends RpcOptVal<Boolean>
+      implements ObjectTargetOpt {
+
+    private static final long serialVersionUID = -7764590745622588287L;
+
+    private OverrideUnlockedRetention(boolean val) {
+      super(StorageRpc.Option.OVERRIDE_UNLOCKED_RETENTION, val);
+    }
```

Comment:
same here, erroring mapper for updateObject() method similar to that of matchGlob